tools/packaging/packaging.psm1

```diff
@@ -299,7 +300,36 @@ function Start-PSPackage {
                     New-PdbZipPackage @Arguments
                 }
             }
+            "min-size" {
+                # Remove symbol files, xml document files.
+                Remove-Item "$Source\*.pdb", "$Source\*.xml" -Force
 
+                if ($Environment.IsWindows) {
```

Comment:
Here we use `$Environment.IsWindows` but below we use the builtin `$IsLinux`. Should they be consistent with each other or is there a reason they're different?

Reply:
When building for Linux packages, we are always on pwsh, where `$IsLinux` is available, but when building windows, it looks to me we are building using Windows PowerShell. I did try using `$IsWindows` here, but failed and thus used `$Environment.IsWindows` like in other places in this file.

Reply:
I changed `$IsLinux` to `$Environment.IsLinux`.